.github/workflows/ci.yml

```
@@ -0,0 +1,51 @@
+  pull_request:
+    branches:
+      - dev
+
+jobs:
+  test:
+    name: run test code on gradle
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)